guessTheNumber.js

```diff
@@ -0,0 +1,19 @@
+var playerNumber = prompt("I'm thinking of a number between 1 and 100. Can you guess the number I'm thinking of?")
+
+min = 1
+max = 100
+var randomize =  Math.random() * (max-min) + min;
+var ourNumber  = Math.round(randomize)
+
+while(ourNumber >= max) {
```

Comment:
These `while` loops are not necessary - `ourNumber` should already be within the range of `min` and `max` based on how it is getting generated.